src/components/sbb-timetable-row/sbb-timetable-row.sample-data.js

```diff
@@ -203,7 +203,9 @@ export const disturbanceTrip = {
   notices: [
     {
       name: 'R',
-      text: 'Platzreservierung möglich',
```

Comment:
Our example data should be in English language, but I see there is a lot more, so you can leave it for the moment.